src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -213,6 +213,11 @@ public struct Configuration
             internal bool ConvertClassicalControl =>
                 this.RuntimeCapability != null && this.RuntimeCapability == RuntimeCapability.BasicMeasurementFeedback;
 
+            /// <summary>
+            /// Indicates whether the compiler will remove lambda expressions and replace them with calls to generated callables.
+            /// </summary>
+            internal bool LiftLambdaExpressions => true; // ToDo: replace this with the actual condition.

```

Comment:
I'm not actually sure what the condition should be for the Lift Lambda Expressions rewrite step should be. @bettinaheim